Tests/MountebankSwiftTests/Api/Endpoint/EndpointTests.swift

```diff
@@ -0,0 +1,61 @@
+import XCTest
+@testable import MountebankSwift
+
+final class EndpointTests: XCTestCase {
+
+    // swiftlint:disable:next force_unwrapping
+    private static let exampleBaseUrl = URL(string: "https://localhost")!
+
+    func testGetAllImposters() {
+        [
+            makeUrlFromEndpoint(.getAllImposters(
+                parameters: ImposterParameters()
+            )): "\(Self.exampleBaseUrl.absoluteString)/imposters",
+            makeUrlFromEndpoint(.getAllImposters(
+                parameters: ImposterParameters(replayable: true)
+            )): "\(Self.exampleBaseUrl.absoluteString)/imposters?replayable=true",
+            makeUrlFromEndpoint(.getAllImposters(
+                parameters: ImposterParameters(removeProxies: false)
+            )): "\(Self.exampleBaseUrl.absoluteString)/imposters?removeProxies=false",
+        ].forEach { endpoint, expectedResult in
+            XCTAssertEqual(endpoint, URL(string: expectedResult))
+        }
```

Comment:
nit: test both params together resulting in 

`/imposters?replayable=true&removeProxies=false`